classes/helper/HelperOptions.php

```diff
@@ -306,6 +313,11 @@ public function displayOptionTypeDisabled($key, $field, $value)
     public function getOptionValue($key, $field)
     {
         $value = Tools::getValue($key, Configuration::get($key));
+
+        if (!empty($field['skipIsCleanHtml'])) {
```

Comment:
```suggestion
        if (!empty($field['skip_clean_html'])) {
```
The convention for legacy classes configuration is more snake case, so we should follow the same for this part of the code I think